### PR TITLE
Updating errata design

### PR DESCRIPTION
**Pull request by sboldyreva**
These are proposals to update current errata design to match AlmaLinux wiki/brandbook.
1) Main page:
* Topbar:
  * Changed logo to icon+text as it's in AlmaLinux Wiki
  * Added buttons tht lead to the website/github/wiki/chat as it is in AlmaLinux Wiki
* Changed font to Monstserrat
2) Errata template:
* Replaces background wallpaper picture that is generated and cropped in different ways to the backgroung color from the brandbook
* Rounded corners 
* Adjusted width to 70% for better look. But I'm not sure how this change would impact responsive design (on a phone/tablet) so definetly need a closer look here.
* Added hover to the logo on the topbar so it's more obvious that it actually leads somewhere 

Ideally I would replace the template's topbar to the same manner as I did with the main page and add a "back" feature for users to easily navigate back to the main page. For these changes I'll need the developers' help which files I need to work in.

### CODE REVIEW

**Review by Korulag — APPROVED** [review]

File: public/assets/images-style.css [hunk 1]
```diff
@@ -28,8 +28,11 @@ a:hover {
     margin-top: 0.65rem;
     margin-left: -1rem;
     vertical-align: top;
-	width: 160px;
-	
+	 width: 160px;
```
Comment (javihernandez) [inline public/assets/images-style.css]: I see some weird indentations around, some use tabs, other use spaces. While you're around, can you please make all them consistent?

File: public/assets/images-style.css [hunk 2]
```diff
@@ -128,4 +134,9 @@ a:hover {
 
 .checksum {
     color: #666666;
-}
\ No newline at end of file
+}
+.row {
+    max-width: 70%;
```
Comment (javihernandez) [inline public/assets/images-style.css:133]: I get a blank space right after the page footer, but if you tested on different browsers with different layouts/resolutions and it's fine, I'm fine with it too.

**Review by javihernandez — COMMENTED** [review]
Thanks for this update @sboldyreva!
Just a suggestion, can we get a small addition in the README.md file that makes reference to the AlmaLinux BrandBook and that any style changes must follow such guidelines?

File: public/assets/images-style.css (hunk 2 above)
Comment (sboldyreva) [reply public/assets/images-style.css:133]: @javihernandez can you get me a screenshot?

File: public/assets/images-style.css (hunk 1 above)
Comment (sboldyreva) [reply public/assets/images-style.css]: please review

**Comment by sboldyreva** [comment]
> Thanks for this update @sboldyreva! Just a suggestion, can we get a small addition in the README.md file that makes reference to the AlmaLinux BrandBook and that any style changes must follow such guidelines?

added

File: public/assets/images-style.css (hunk 2 above)
Comment (javihernandez) [reply public/assets/images-style.css:133]: image

I'm on a 14' laptop at 1920x1080 just in case

**Review by anfimovdm — APPROVED** [review]

**Review by javihernandez — APPROVED** [review]

**Pull request merged**